src/config/index.js

```diff
@@ -1,16 +1,16 @@
 export const API = Object.freeze({
-  HOST: "http://18.188.64.13",
-  BROKER_NODE_A: "http://18.188.64.13",
-  BROKER_NODE_B: "http://18.188.230.212",
+  HOST: "http://ALPHA_BROKER_IP",
+  BROKER_NODE_A: "http://ALPHA_BROKER_IP",
+  BROKER_NODE_B: "http://BETA_BROKER_IP",
```

Comment:
We should separate these IP configuration options into a different file at some point and add it to .gitignore.

Reply:
They should be in the .env file, which i think is already ignored.